README.md

```diff
@@ -3,10 +3,9 @@ NovelRT
 
 NovelRT is a cross-platform, flexible Visual Novel and 2D game engine. It is aimed at designers and developers alike, however many of the designer tools and features we have on our roadmap have yet to be implemented.
 
-| Job | Debug Status | Release Status |
-| --- | ------------ | -------------- |
-| Windows x64 | [![Build Status](https://github.com/novelrt/NovelRT/workflows/Windows%20x64%20-%20Debug/badge.svg?event=schedule)](https://github.com/novelrt/NovelRT/actions?query=workflow%3A%22Windows+x64+-+Debug%22) | [![Build Status](https://github.com/novelrt/NovelRT/workflows/Windows%20x64%20-%20Release/badge.svg?event=schedule)](https://github.com/novelrt/NovelRT/actions?query=workflow%3A%22Windows+x64+-+Release%22) |
-| Ubuntu x64 | [![Build Status](https://github.com/novelrt/NovelRT/workflows/Ubuntu%20x64%20-%20Debug/badge.svg?event=schedule)](https://github.com/novelrt/NovelRT/actions?query=workflow%3A%22Ubuntu+x64+-+Debug%22) | [![Build Status](https://github.com/novelrt/NovelRT/workflows/Ubuntu%20x64%20-%20Release/badge.svg?event=schedule)](https://github.com/novelrt/NovelRT/actions?query=workflow%3A%22Ubuntu+x64+-+Release%22) |
+[![Latest Nightly Build](https://img.shields.io/github/workflow/status/novelrt/NovelRT/Continuous%20Integration%20v2/main?label=Nightly%20Build&style=plastic)](https://github.com/novelrt/NovelRT/actions/workflows/build-system.yml)
+
+[![Latest Release](https://img.shields.io/github/v/release/novelrt/NovelRT?color=blueviolet&include_prereleases&label=Latest%20Release&style=plastic)](https://github.com/novelrt/NovelRT/releases)
```

Comment:
We lose 4 individual "workflow" badges, so I added two specific ones here that are customisable somewhat - one for the nightly builds, and one for releases.

Reply:
We should probably drop include_prereleases here and only show the latest *stable* release - the release pipeline in Fabulist marks releases as prerelease because it's totally possible that a `main` build could have bugs in it, and it's intended that after some human intervention we will manually mark releases as stable.